src/blob/core.rs

```diff
@@ -323,8 +327,12 @@ where
         }))
     }
 
-    pub(crate) async fn mark_all_as_deleted(&mut self, key: &K) -> Result<Option<u64>> {
-        if self.index.get_any(key).await?.is_some() {
+    pub(crate) async fn mark_all_as_deleted(
```

Comment:
You've changed that function in https://github.com/qoollo/pearl/pull/209. Probably, you should merge https://github.com/qoollo/pearl/pull/209 into this branch

Reply:
Sure, once #209 is ready I'll merge it here